surveys/sphinx-survey-2025.yml

```diff
@@ -203,37 +233,38 @@ questions:
   - "Stability"
   - "Variety of features"
   - "Other projects in my ecosystem were using it"
-  - "It was chosen by others on the project"
+  - "It was chosen by others or before I joined"
 
-# Motivation: Use answer to branch survey questions. 
+# BRANCH POINT
 - title: >-
     Do you use Sphinx?
     (Whether working on the project, using it for another project,
     or developing Sphinx-extensions)
   type: "yes_no_jump"
   ref: "use_sphinx"
-  if_no_jump_to: arent_sphinx
-  otherwise_jump_to: sphinx_years
+  if_no_jump_to: "no_sphinx"
+  otherwise_jump_to: "yes_sphinx"
 
 # Motivation: Gather information on possible motivations for documentation tool selection.
 - title: >-
-    If you aren’t using Sphinx for your documentation, please explain why
-    you chose the one you are using
-  ref: "arent_sphinx"
+    If you don’t use Sphinx for your documentation,
+    please explain why you chose the one you are using
+  ref: "no_sphinx"
   type: long_text
   always_jump_to: the_end
 
 # Motivation: Understand the responder's self-reported background and skills.
 - title: "How many years have you been using Sphinx?"
-  ref: sphinx_years
+  ref: "yes_sphinx"
   type: number
 
 # Motivation: Explore patterns in what people using Sphinx value in it.
 - title: "What is the main reason you use Sphinx?"
   other: true
   choices:
   - "API documentation generation from Python docstrings"
```

Comment:
The fact that it's open source is also an important consideration in corporate settings. pigweed.dev explicitly opted for an OSS solution even though closed source solutions also exist within Google